packages/odyssey-react/src/components/Table/Table.tsx

```diff
@@ -47,11 +53,9 @@ export type Props = {
   * Provides users of assistive technologies with context for the table contents
   */
   caption: string
-} & ContainerProps & ComponentProps<'table'>
-
-export type Ref = HTMLTableElement;
+} & ContainerProps & ComponentPropsWithRef<'table'>
```

Comment:
@andrewberg-okta apologies if I'm missing something obvious here - but is there any reason to have multiple prop shape annotations  here all combined via intersection `Props & ContainerProps`? Or can this just be simplified like this:
```
interface Props extends ComponentPropsWithRef<'table'> {
  // ....
}
```